persistence/src/main/java/fi/otavanopisto/pyramus/domainmodel/users/StaffMember.java

```diff
@@ -82,4 +97,9 @@ public void setProperties(Map<String, String> properties) {
   @Column (name = "value", length = 255)
   @CollectionTable (name = "StaffMemberProperties", joinColumns = @JoinColumn(name = "staffMember_id"))
   private Map<String, String> properties = new HashMap<String, String>();
+
+  @ManyToMany (fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+  @JoinTable (name = "__StaffMemberStudyProgrammes", joinColumns = @JoinColumn(name = "staffMember"), inverseJoinColumns = @JoinColumn(name = "studyProgramme"))
```

Comment:
I would prefer table name to not contain underscores, at least mysql handles them weirdly in some cases.

Reply:
This has been the naming convention for all tables that are not represented by a dedicated entity class. There have been 12 such tables before this one, so for consistency's sake it would be better to keep this one as-is. Perhaps all these 13 intermediary tables could be renamed later in some separate maintenance branch?